CONTRIBUTING.md

```diff
@@ -35,6 +35,15 @@ goreleaser --snapshot --skip-publish --rm-dist
 
 ## Conventions
 
+### Command usage
+
+Command usage descriptions (namely `cobra.Command.Use`) should follow the [docopt](http://docopt.org/) syntax.
+
+Additionally:
+- Optional arguments should be the first in the usage string.
```

Comment:
Sorry, you're right, I meant flags there. Docopt doesn't say anything about argument/flag order. Having optional flags first, then arguments and then required flags made the most sense to me because then the ordering would be most specific to least specific.

Reply:
About the ordering, this is what the CLI Guidelines have to say: https://clig.dev/#help
They suggest putting the `[options]` first as well (like jq and git do it too for example). But generally they put all flags first (even required ones) and the positional arguments last, while we do optional flags < positional args < required flags. Maybe we can move the positional args all the way to the back like jq does. 
On the other hand, if you look [here](https://github.com/hetznercloud/cli/pull/709/files#diff-fd9e246bf99bd641dacfa0eb236373abe4720c1398ad0a468266b3357c92b845R20) for example, positional args would be easily overlooked if they are placed last. What's your opinion on this?

Reply:
I would always try to have the required and optional flags next to each other, having the arguments in the middle seem weird.

Whether we put the arguments at the beginning or the end, I think I prefer at the beginning. When I write a command, I prefer to go in order of importance `<resource type> <actions> <resource id> <resource fields (e.g. labels)>`. Having the flags before also seem weird.

Reply:
I do not have a strong opinion on this.

While using the CLI I use the same mental model as @jooola and also put `<args> <req flags> <optional flags>`.

Reply:
It looks like you only committed some of the changes, there are still many files with old order, for example `internal/cmd/loadbalancer/update_service.go` or `internal/cmd/firewall/remove_from_resource.go`